database/sdc_dummydatagenerator.js

```diff
@@ -10,18 +10,26 @@ const getRandomInt = (max) => {
   return Math.ceil(Math.random() * Math.floor(max));
 }
 
+const cards = ["Breakfast", "Lunch", "Dinner", "Brunch", "Prixfixe", "Set Menu"]
+
+const getRandomCard = () => {
+  return cards[Math.floor(Math.random() * cards.length)]
+}
+
 //////////////////////////
 // Dummy Data Generator //
 //////////////////////////
 
 var NUM_OF_RECORDS = 10000000
 
-const wstream = fs.createWriteStream(__dirname + '/sdc_dummydata.txt');
+const wstream = fs.createWriteStream(path.join(__dirname, '/data.csv'), { flags: 'w' });
```

Comment:
Johnson mentioned that the { flags: 'w' } might not be necessary. Just something to keep in mind.